{"name": "Ошибка при повторном принятии заказа", "status": "passed", "steps": [{"name": "Принять заказ №455120 курьером 507921", "status": "passed", "parameters": [{"name": "order_id", "value": "455120"}, {"name": "courier_id", "value": "507921"}], "start": 1745261080344, "stop": 1745261080427}, {"name": "Принять заказ №455120 курьером 507921", "status": "passed", "parameters": [{"name": "order_id", "value": "455120"}, {"name": "courier_id", "value": "507921"}], "start": 1745261080427, "stop": 1745261080463}], "attachments": [{"name": "log", "source": "8e1dc6d4-41b3-472a-8234-b0c247f9b781-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "7e624109-59aa-4101-b4a1-54069f38ad7d-attachment.txt", "type": "text/plain"}], "start": 1745261080344, "stop": 1745261080464, "uuid": "7bd6973a-7408-40e8-94b0-4d361e1ab96c", "historyId": "c1197f695c09076c6067f2fd00527008", "testCaseId": "c1197f695c09076c6067f2fd00527008", "fullName": "tests.test_additional.TestOrderAcceptance#test_accept_order_already_accepted", "labels": [{"name": "epic", "value": "API Яндекс.Самокат"}, {"name": "story", "value": "Принятие заказа"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_additional"}, {"name": "subSuite", "value": "TestOrderAcceptance"}, {"name": "host", "value": "Gemini_78"}, {"name": "thread", "value": "2332-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_additional"}]}